src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,252 @@
+import {
+  Box,
+  chakra,
+  ChakraComponent,
+  useMultiStyleConfig,
+} from "@chakra-ui/react";
+import React, { forwardRef, useState } from "react";
+
+import Button from "../Button/Button";
+import Heading, { HeadingSizes } from "../Heading/Heading";
+import Icon, { IconProps } from "../Icons/Icon";
+
+export const bannerTypesArray = [
+  "informative",
+  "negative",
+  "neutral",
+  "positive",
+  "recommendation",
+  "warning",
+] as const;
+export type BannerTypes = typeof bannerTypesArray[number];
+export const bannerBgColorsArray = [
+  "brand.primary-05",
+  "section.blogs.primary-05",
+  "section.books-and-more.primary-05",
+  "section.connect.primary-05",
+  "section.education.primary-05",
+  "section.locations.primary-05",
+  "section.research.primary-05",
+  "section.research-library.lpa-05",
+  "section.research-library.schomburg-05",
+  "section.research-library.schwarzman-05",
+  "section.whats-on.primary-05",
+  "dark.brand.primary-05",
+  "dark.section.blogs.primary-05",
+  "dark.section.books-and-more.primary-05",
+  "dark.section.connect.primary-05",
+  "dark.section.education.primary-05",
+  "dark.section.locations.primary-05",
+  "dark.section.research.secondary-05",
+  "dark.section.research-library.lpa-05",
+  "dark.section.research-library.schomburg-05",
+  "dark.section.research-library.schwarzman-05",
```

Comment:
@bigfishdesign13 the spec doc has these light/dark colors as `dark.section.research-library-lpa.primary-05`. But in the colors file, the design token value is `dark.section.research-library.lpa-05`. So there are some inconsistencies. I updated this to reflect the current design tokens in the repo but we can go either way: update the design token values or update the spec doc.

Reply:
I will update the spec doc.  That is an error on my part.